src/pretix/presale/views/event.py

```diff
@@ -104,6 +108,29 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class EventIcalDownload(EventViewMixin, View):
+    def get(self, request, *args, **kwargs):
+        if not self.request.event:
+            raise Http404(_('Unknown event code or not authorized to access this event.'))
+
+        cal = Calendar()
+        cal.add('version', '2.0')
+        cal.add('prodid', '-//{}//{}//'.format(self.request.event.organizer.name, self.request.event.name))
+
+        event = Event()
+        event.add('summary', self.request.event.name)
+        event.add('dtstart', self.request.event.date_from)
```

Comment:
I set up 4 different events for testing the above feature.
All events start at 2017-02-24 00:00:00 (Asia/Singapore, GMT+8) and end at 2017-02-26 00:00:00 (Asia/Singapore, GMT+8).

Since I am living in Japan, the calendar is using GMT+9.
I also attached the .ics files in case you want to import it to somewhere else.

**2-DAY-EVENT:** normal event which lasts for 2 days
```
DTSTART;TZID=Asia/Singapore;VALUE=DATE-TIME:20170224T000000
DTEND;TZID=Asia/Singapore;VALUE=DATE-TIME:20170226T000000
```
![2-day-event](https://cloud.githubusercontent.com/assets/25281597/23222253/a71dd67c-f96a-11e6-9dc4-f4d823915b6e.png)
**2-DAY-EVENT-NO-TIME:** same event but time is not shown ---> full day event
```
DTSTART;VALUE=DATE:20170224
DTEND;VALUE=DATE:20170226
```
![2-day-event-no-time](https://cloud.githubusercontent.com/assets/25281597/23222251/a717028e-f96a-11e6-9831-7238232faf33.png)
**2-DAY-EVENT-NO-ENDATE:** same event but end date is not shown ---> event with same start & end time
```
DTSTART;TZID=Asia/Singapore;VALUE=DATE-TIME:20170224T000000
```
![2-day-event-no-enddate](https://cloud.githubusercontent.com/assets/25281597/23222250/a711f6f4-f96a-11e6-89a6-8e192499041d.png)
**2-DAY-EVENT-NO-ENDATE-AND-TIME:** same event but time & end date are not shown 
```
DTSTART;VALUE=DATE:20170224
```
![2-day-event-no-enddate-and-time](https://cloud.githubusercontent.com/assets/25281597/23222252/a71713be-f96a-11e6-9ff8-4d4f2f28f304.png)


**Downloads:**
[event-ics.zip](https://github.com/pretix/pretix/files/794216/event-ics.zip)

Please let me know if you find any problem :)
I added a new commit for the above changes!